build_poh.sh

```diff
@@ -5,7 +5,7 @@ set -e
 
 command_exists () { type "$1" &> /dev/null ; }
 
-requirements="java mvn git docker-compose mysql zip tar"
+requirements="java mvn git docker-compose mysql zip tar wget xargs"
```

Comment:
Please add also `mysqldump`. It would be nice also to show the standard output from the OS, e.g.:
```
Command 'mysqldump' not found, but can be installed with:

sudo apt install mysql-client-5.7
sudo apt install mariadb-client-10.1
```